src/PendingShortScheduleCommand.php

```diff
@@ -23,6 +23,8 @@ class PendingShortScheduleCommand
 
     protected array $constraints = [];
 
+    protected ?string $basePath = null;
```

Comment:
Why is this optional? Doesn't `base_path()` always return a string?

Reply:
Yes but I set it null as it was originally. I only set the base path if using the command() method as stated in the PR.

Reply:
https://github.com/spatie/laravel-short-schedule/pull/9/files#diff-0c85a47633423834140871067a319ffdR43 I did not set base path during the exec() command because I assumed there was a reason you felt it did not need setting in the first place.

Reply:
Right, thanks for explaining.

Would it make sense to always pass `base_path()` as the second argument of `fromShellCommandline`? If that would work, we wouldn't even need an instance variable.

Reply:
It is tricky. We could do that and happy to make that change. Another suggestion from my co-worker this afternoon was to have a new method basePath() that could be chained. So that it could be over ridden if the situation arose.

If you run it from the CLI manually eg php artisan short-schedule:run the $cwd inside the process call is the base path of laravel already. Because it's where its called from. 

My thought was only only calls to artisan commands require the $cwd to be set and any exec commands do not. 

If someone else needs to set it they could open a PR adding a method to set the property. My intention is only to get the short-schedule to work under supervisor correctly resolving the previously reported issue.

Reply:
Let's try the simplest solution first. If it works in all situations by passing `base_path` as the second argument of `fromShellCommandline`, then I think that would be the best solution.

Reply:
Pushed

Reply:
@freekmurze github seems to be stuck. It's not running that prefer-lowest test. I don't know why. Are you able to make it retry?